app/code/Magento/CatalogInventory/etc/adminhtml/system.xml

```diff
@@ -13,7 +13,7 @@
             <resource>Magento_CatalogInventory::cataloginventory</resource>
             <group id="options" translate="label" type="text" sortOrder="1" showInDefault="1" showInWebsite="1" showInStore="1">
                 <label>Stock Options</label>
-                <field id="can_subtract" translate="label" type="select" sortOrder="2" showInDefault="1" showInWebsite="0" showInStore="0" canRestore="1">
+                <field id="can_subtract" translate="label" type="select" sortOrder="2" showInDefault="1" showInWebsite="1" showInStore="0" canRestore="1">
```

Comment:
We can not change legacy(core) code
We need to override this option in new config in new `InventorySales` module